index.js

```diff
@@ -2,13 +2,16 @@
 
 module.exports = toMDAST;
 
-var minify = require('rehype-minify-whitespace')();
+var rehypeMinifyWhitespace = require('rehype-minify-whitespace');
 var xtend = require('xtend');
 var one = require('./lib/one');
 var handlers = require('./lib/handlers');
 
 function toMDAST(tree, options) {
   var settings = options || {};
+  var minify = rehypeMinifyWhitespace({
+    newlines: typeof settings.newlines === 'boolean' ? settings.newlines : false
```

Comment:
Could we just do `settings.newlines === true` here? That does the same right, and is a bit more readable?